Exercise_2.py

```diff
@@ -1,22 +1,50 @@
+# Time Complexity : O(n)
```

Comment:
The better way to implement this would be to add and remove from the head, bringing down the Time complexity to O(1).

Reply:
Noted